include/stdlib.h

```diff
@@ -64,9 +64,11 @@ __Z_EXPORT char* getenv(const char*) __asm("@@A00423");
 extern "C" {
 #endif
 /**
- * C mkdtemp implementation
+ * C Lib functions that do not conflict with z/OS LE
  */
 __Z_EXPORT char *mkdtemp(char *templ);
+__Z_EXPORT int getloadavg(double loadavg[], int nelem);
+__Z_EXPORT const char * getprogname(void);
```

Comment:
can you confirm that the couple of programs that pull in their own getprogname won't have any signature conflicts with this prototype? These are the ones I know of: https://github.com/search?q=org%3AZOSOpenTools%20getprogname&type=code

Reply:
Thanks, I'll test them

Reply:
Tests coreutils, diffutils, and man-db. No regressions found